lib/staccato/V4/tracker.rb

```diff
@@ -173,6 +182,10 @@ def user_id
       nil
     end
 
+    def timestamp_micros
+      nil
+    end
```

Comment:
I might be missing something, but this just looks like the method's defined twice in the class.

Reply:
This one is for `NoopTracker` class.

File has 2 classes defined. One is `Tracker` and another one is `NoopTracker`.

Reply:
Thanks, missed that before.